service/controller/aws/v17/version_bundle.go

```diff
@@ -8,9 +8,14 @@ func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
 			{
-				Component:   "TODO",
-				Description: "TODO",
-				Kind:        versionbundle.KindAdded,
+				Component:   "node-exporter",
+				Description: "Disabled ipvs collector.",
```

Comment:
One is imperative, one is simple past. I think we should go consistently with imperative speech here.

Reply:
Should be fixed if i remember my english lessons well enough, ptalaaaa

Reply:
I meant to post here :(

I think imperative is correct but the docs are outdated. Discussing grammar in PRs is something I'd like to avoid ;)

https://github.com/giantswarm/fmt/blob/master/releases/how_write_changelog_message.md